app/views/hyku/api/v1/work/_work.json.jbuilder

```diff
@@ -14,6 +14,8 @@ json.buy_book work.try(:buy_book)
 json.challenged work.try(:challenged)
 json.cname @account.cname
 #                                         "collections" => nil,
+json.contributor work.contributor
+json.creator work.creator
```

Comment:
Do these need to be `contributor_display` and `creator_display` instead?  `contributor` and `creator` hold the JSON blob.

Reply:
Oooop. Ill check

Reply:
`creator_display` is not a method. We can do `JSON.parse(creator)`